spec/lib/gakubuchi/template_spec.rb

```diff
@@ -101,6 +111,14 @@
 
       it { is_expected.to be_an_instance_of Pathname }
       it { is_expected.to be_fnmatch(expectation) }
```

Comment:
https://github.com/yasaichi/gakubuchi/pull/11#discussion_r101890966 refers to them and the others of your changes on this spec file ... 😢 
Could you modify all of them for consistency?